src/components/maps/RarityMapInfo.tsx

```diff
@@ -0,0 +1,278 @@
+            target="_blank"
+          >
+            source
+          </Link>
+        </li>
+        <li>
+          A well-balanced ecosystem maintains the health of the environment. This ensures that human
+          beings have access to clean air and water, and fertile land for agriculture.{" "}
+          <Link
+            as={CalciteLink}
+            href="https://www.gviusa.com/blog/why-should-we-save-endangered-species/"
+            rel="noopener noreferrer"
+            target="_blank"
+          >
+            source
+          </Link>
+        </li>
+        <li>
+          Each living thing contains a unique reservoir of genetic material that has evolved over
+          eons. This material cannot be retrieved or duplicated if lost.{" "}
+          <Link
+            as={CalciteLink}
+            href="https://www.worldanimalfoundation.org/advocate/wild-earth/params/post/1285404/why-save-endangered-species"
+            rel="noopener noreferrer"
+            target="_blank"
+          >
+            source
+          </Link>
+        </li>
+      </ul>
+    </InfoItem>
+
+    <InfoItem>
+      <h3>
+        <b>Our Spatial Analysis Workflow</b>
+      </h3>
+      <p>
+        <ol>
+          <li>
+            <p>
+              Acquired the protected areas and marine species rarity scores within the Gulf of
+              Mexico, using the
+              <b>Overlay Layers</b> tool on the input data (see below for the sources of the input
+              data): a layer of worldwide protected areas, a grid cell layer with marine species
+              rarity scores, and the Gulf of Mexico boundary.
+            </p>
+          </li>
+          <li>
+            <p>
+              Considering that we used the gird cells as the unit of analysis, and since some
+              protected areas spread across multiple cells, called <b>Overlay Layers</b> again to
+              split the protected areas by cells.
+            </p>
+          </li>
+          <li>
+            <p>
+              Dissolved the protected areas that were within the same cell, using{" "}
+              <b>Dissolve Boundaries.</b>
+            </p>
+          </li>
+          <li>
+            <p>
+              Used the <b>Summarize Within</b> tool to generate the result layer that includes both
+              the portion of areas that have been marked as protected area, and marine species
+              rarity score, within each grid cells.
+            </p>
+          </li>
+          <li>
+            <p>
+              Added the result layer to a map and configured symbology to show the relationship
+              between the protected propertion and marine species rarity score.
+            </p>
+          </li>
+        </ol>
+      </p>
+    </InfoItem>
+    <InfoItem>
+      <h3>
+        <b>Flowchart</b>
+      </h3>
+      <img
+        alt="Flowchart"
+        height="300"
+        src="public/img/spatial_analysis_flowchart_rarity_protection.png"
```

Comment:
```suggestion
        src="public/img/spatial_analysis_flowchart_rarity_protection.png"
```
src={process.env.PUBLIC_URL + '/img/spatial_analysis_flowchart_rarity_protection.png'}